experimental/go-modules/templates/default/go.mod

```diff
@@ -0,0 +1 @@
+module github.com/example
```

Comment:
Hopefully obvious to most go developers, but maybe a comment saying that this example value should be changed would be helpful?